tests/components/tuya/fixtures/dlq_dikb3dp6.json

```diff
@@ -0,0 +1,148 @@
+{
```

Comment:
@rokam I missed this during review.

New fixture should be added to `__init__.py`

https://github.com/home-assistant/core/blob/270a9a5a98cae57542f39f4b6ff7b05e8eb9b9ba/tests/components/tuya/__init__.py#L116-L121

Can you please create a follow-up PR?